modules/gmake/tests/proj/proj_configurations_tests.lua

```diff
@@ -0,0 +1,345 @@
+local gmake = require('gmake')
+local proj = gmake.proj
+
+local GmakeProjConfigurationsTests = test.declare('GmakeProjConfigurationsTests', 'gmake-proj', 'gmake')
```

Comment:
For the VS and Xcode exporters, I'm breaking out the unit tests suites for each method (one for `project.configTargetDir()`, a separate suite for `proj.configIntermediateDir()`, etc.). You already did that for most of the other ones; not sure why these are lumped together?